MapboxNavigation/NavigationMapView.swift

```diff
@@ -324,6 +324,7 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
     }
     
     @objc private func disableUserCourseTracking() {
+        guard tracksUserCourse else { return }
```

Comment:
`disableUserCourseTracking()` is called on every map move. We don't need to continually call didSet on `tracksUserCourse`.